src/human_encoding/parse/mod.rs

```diff
@@ -605,8 +605,8 @@ mod tests {
                 assert_eq!(main.cmr().to_string(), cmr);
 
                 let witness: HashMap<_, _> = witness
-                    .into_iter()
-                    .map(|(name, value)| (Arc::from(*name), Value::power_of_two(*value)))
+                    .iter()
```

Comment:
7202f933fddd92fc119835292d7f141de3d0d8c2: For some reason this does not trigger a clippy lint in Rust 1.74.1 (Linux)